src/brogue/Monsters.c

```diff
@@ -89,6 +89,13 @@ creature *generateMonster(short monsterID, boolean itemPossible, boolean mutatio
     }
 
     prependCreature(monsters, monst);
+    initializeMonster(monst, itemPossible);
+
+    return monst;
+}
+
+void initializeMonster(creature *monst, boolean itemPossible) {
```

Comment:
Could you add a comment describing what kind of state the passed monster is allowed/expected to be in? Seems like we have multiple levels initialised now and would be good to track

Reply:
Sure thing. Done.